apis/generate.go

```diff
@@ -30,9 +30,9 @@ limitations under the License.
 //go:generate go run -tags generate sigs.k8s.io/controller-tools/cmd/controller-gen object:headerFile=../hack/boilerplate.go.txt paths=./pkg/meta/... crd:crdVersions=v1 output:artifacts:config=../docs/api-docs/crds
 
 // Generate clientset for types.
-//go:generate rm -rf ../internal/client
-//go:generate go run -tags generate k8s.io/code-generator/cmd/client-gen --clientset-name "versioned" --build-tag="ignore_autogenerated" --go-header-file "../hack/boilerplate.go.txt" --output-package "github.com/crossplane/crossplane/internal/client/clientset" --input-base "github.com/crossplane/crossplane/apis" --output-base "../tmp-clientgen" --input "pkg/v1alpha1,pkg/v1beta1,pkg/v1,apiextensions/v1"
-//go:generate cp -r ../tmp-clientgen/github.com/crossplane/crossplane/internal/client ../internal/client
+//go:generate rm -rf ../pkg/client
```

Comment:
If we do expose these clients, does it make sense to drop the `pkg` directory and have their import path be `github.com/crossplane/crossplane/clients`? The `pkg` directory has questionable value - see https://github.com/golang-standards/project-layout/issues/10 for some debate around it.